src/unicef_vision/synchronizers.py

```diff
@@ -143,9 +143,7 @@ def __init__(self, business_area_code=None, *args, **kwargs):
 
         self.filename = filename
         self.business_area_code = business_area_code
-        logger.info('Business_area_code is {}'.format(business_area_code))
-
-        super().__init__(business_area_code, *args, **kwargs)
```

Comment:
@domdinicola i removed it because passing any argument to the superclass constructor throws an error. Not sure how this error slipped in.. maybe this specific syncronizer was never used.

Reply:
@csabadenes I still think we need this....

Reply:
I have retested it, when i restored that code i have this error:
`super().__init__(business_area_code, *args, **kwargs) TypeError: object.__init__() takes no parameters`
Maybe i am missing something, but i can't see how that line of code could work. The superclass does not have a custom constructor.

Reply:
I have restored that line of code, i can remove it again if needed.